dynamic_rest/serializers.py

```diff
@@ -346,6 +361,22 @@ def get_fields(self):
         for name in deferred:
             serializer_fields.pop(name)
 
+        # Set read_only flags based on read_only_fields meta list.
+        # Here to cover DynamicFields not covered by DRF.
+        ro_fields = getattr(self.Meta, 'read_only_fields', [])
+        self.make_fields_read_only(serializer_fields, ro_fields, True)
+
+        # Toggle read_only flags for immutable fields.
+        immutable_field_names = self._get_flagged_field_names(
+            serializer_fields,
+            'immutable'
+        )
+        self.make_fields_read_only(
```

Comment:
This could potentially set a field that is flagged read-only AND immutable as not read-only.

I'm not sure why you would ever do that in practice, but I can see it happening accidentally from an overdose of serializer mixins.